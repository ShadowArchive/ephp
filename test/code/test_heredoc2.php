<?php

$data = <<<EOF

Numbers: \x30 \x31
Octals: \41 \42
Symbols: $ ! #
EOF;

print "$data\n";

