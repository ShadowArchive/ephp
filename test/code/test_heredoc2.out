
Numbers: 0 1
Octals: ! "
Symbols: $ ! #
